### ISSUE TITLE
Deepspeech German fails to create model

### ISSUE THREAD

**Issue by maxbachmann** [rhasspy/rhasspy#100]
When using deepspeech in german I get the following exception while creating the model:

```bash
ERROR: Model provided has model identifier 'inpu', should be 'TFL3'

Error at reading model file /profiles/de/deepspeech/model/0.6.1/output_graph.pb
[ERROR:2020-08-18 20:12:07,451] rhasspyasr_deepspeech_hermes: transcribe
Traceback (most recent call last):
  File "/usr/lib/rhasspy/rhasspy-asr-deepspeech-hermes/rhasspyasr_deepspeech_hermes/__init__.py", line 319, in transcribe
    transcription = self.transcriber.transcribe_wav(wav_bytes)
  File "/usr/lib/rhasspy/rhasspy-asr-deepspeech/rhasspyasr_deepspeech/transcribe.py", line 42, in transcribe_wav
    self.maybe_load_model()
  File "/usr/lib/rhasspy/rhasspy-asr-deepspeech/rhasspyasr_deepspeech/transcribe.py", line 139, in maybe_load_model
    self.model = deepspeech.Model(str(self.model_path), self.beam_width)
  File "/usr/lib/rhasspy/.venv/lib/python3.7/site-packages/deepspeech/__init__.py", line 42, in __init__
    raise RuntimeError("CreateModel failed with error code {}".format(status))
RuntimeError: CreateModel failed with error code 12288
```

**Comment by Xalip** [comment]
Which commands did you use for the build process? So I can try to replicate the error.

**Comment by maxbachmann** [comment]
I did not use any command for the build process :sweat_smile:
I simply started rhasspy in german, activated deepspeech in the settings and got this error message as a result. When there is a additional manual step that has to be performed thats probably whats causing my problem.

**Comment by synesthesiam** [comment]
If you're trying this on a Raspberry Pi, that's the problem :/  The German DeepSpeech model does not have a TFLite version, so it only works on x86_64 machines.

**Comment by maxbachmann** [comment]
Ah yes then thats definetly the reason :smile:

It might be cool to hide unsupported options for language/hardware of the user in the settings

**Comment by synesthesiam** [comment]
Will be fixed in 2.5.9 :)
